tests/test_interface.py

```diff
@@ -195,3 +195,21 @@ def test_power_down_can_be_switched(current_value, expected_value):
     interface_state.power_down = current_value
     interface_state.switch_power_down()
     assert interface_state.power_down is expected_value
+
+
+def test_change_callback(capsys):
+    """
+    Test if change_callback function works and assert correct result.
```

Comment:
Ad test: jak se overi, ze callback je opravdu zavolan? Na zacatku v testu vytvaris nejaky stav interface. Callback je skryty pod par metodami, ale zadnou tady nevolas, misto toho volas obycejny `print` (nikoli `test_print`), a overujes, ze `print` vypise to, co mu davas. To on samozrejme vypise. Jen neni otestovano to, ze nejaka metoda s callbackem opravdu vola funkci, kterou jsi ji predala v argumentu.

Reply:
ok zkusím vymyslet :)